src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -700,6 +713,7 @@ export class CalciteColorPicker {
             aria-valuemin="0"
             aria-valuenow={(vertical ? color?.saturationv() : color?.value()) || "0"}
             class={CSS.scope}
+            data-test-id="color-field-scope"
```

Comment:
Can you use the 1st element w/ the scope class or additional class for now? I see the value in this, but would like to spend some time on guidelines on when/how to use it to avoid having test IDs all over.

Reply:
I thought we already accepted this approach in a meeting way back? There are a couple other uses like this if you search for data-test-id.

Reply:
I remember we discussed it, but don't remember adopting it. Feel free to leave this one based on that, but we should revisit since this ends up being included in production code and our build unfortunately doesn't have a way to automatically strip these out.

Reply:
The other uses are in tree/tree-item.

Reply:
Can you create an issue for discussion? If we don't adopt it we can remove both.

Its easier to identify with a data-attribute but I can add an additional class that serves no purpose if needed.